diff -up ./magick/property.c.annotate-fix ./magick/property.c
--- ./magick/property.c.annotate-fix	2017-01-11 09:16:59.390006547 +0100
+++ ./magick/property.c	2017-01-11 09:17:21.634006533 +0100
@@ -3206,8 +3206,6 @@ MagickExport char *InterpretImagePropert
     }
   }
   *q='\0';
-  if (text != (const char *) embed_text)
-    text=DestroyString(text);
   (void) SubstituteString(&interpret_text,"&lt;","<");
   (void) SubstituteString(&interpret_text,"&gt;",">");
   return(interpret_text);
